tools/Dockerfile

```diff
@@ -26,6 +26,12 @@ RUN microdnf update && \
     iputils \
     openssl \
     tar \
+    # util-linux-core contains a basic set of Linux utilities, including
+    # the command logger which allows to enter messages into the system
+    # log.
+    # cpe:2.3:a:kernel:util-linux:*:*:*:*:*:*:*:*
+    # https://nvd.nist.gov/vuln/search/results?isCpeNameSearch=true&query=cpe%3A2.3%3Aa%3Akernel%3Autil-linux%3A*%3A*%3A*%3A*%3A*%3A*%3A*%3A
```

Comment:
I added the link to `util-linux` instead of `util-linux-core` because `util-linux-core` seems not to be an existing product in the NIST NVD.